signal_scanner_bot/env.py

```diff
@@ -42,6 +42,13 @@ def log_vars() -> None:
 TWITTER_ACCESS_TOKEN = _env("TWITTER_ACCESS_TOKEN")
 TWITTER_TOKEN_SECRET = _env("TWITTER_TOKEN_SECRET")
 TRUSTED_TWEETERS = set(str(_env("TRUSTED_TWEETERS", default="")).split(","))
+SEND_HASHTAGS = str(_env("SEND_HASHTAGS", default="")).split(",")
```

Comment:
Per the above note, then we could have a separate convert function for CSV lists and sets!

Reply:
Created _cast_to_list function to resolve this.